src/axom/quest/examples/CMakeLists.txt

```
@@ -118,44 +119,53 @@ if (ENABLE_FORTRAN)
         FOLDER     axom/quest/examples
         )
 
-    blt_add_executable(
-        NAME        quest_inout_interface_F_ex
-        SOURCES     quest_inout_interface.F
-        OUTPUT_DIR  ${EXAMPLE_OUTPUT_DIRECTORY}
-        DEPENDS_ON  ${quest_example_depends} ${quest_example_fortran_depends}
-        FOLDER      axom/quest/examples
-        )
+    # This target fails to compile with hipcc/amdflang on debugs
+    if(NOT ${CMAKE_Fortran_COMPILER} MATCHES "amdflang$" OR NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
```

You have to quote strings that may be empty otherwise you will end up with a very hard to read syntax error here.
```suggestion
    if(NOT "${CMAKE_Fortran_COMPILER}" MATCHES "amdflang$" OR NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```